docs/modules/ROOT/pages/scheduling-configuration.adoc

```diff
@@ -0,0 +1,100 @@
+= Configure scheduling policy for Hazelcast instances
+:description:
+
+Kubernetes allows scheduling policies for the Pods.
+Hazelcast Platform operator also accepts same policies to explicitly allow or disallow certain scheduling policies.
+Detailed documentation about scheduling policies can be found link:https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/[here].
+
+{description}
+
+== Node Selector
+
+The users can specify node selector to define on which nodes Hazelcast instances run.
+The labels in node selector will match the node's label.
+
+For example, we want to run our Hazelcast instances on specific region (e.g us-west1).
+
+The example:
+
+[source,yaml,subs="attributes+"]
+----
+include::ROOT:example$/node-selector.yaml[]
+----
+
+Node selectors are hard requirements. If there is no node to match all of the selectors, then
+the Hazelcast instances will not be scheduled.
+
+More information about node selectors can be found link:https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector[here].
+
+== Node Affinity
+
+The users can specify node affinity to define which nodes will run Hazelcast instances.
+Nodes with specified labels are allowed to run them. Compared to node selectors, node affinity
+allow users to additionally define soft requirements, and more complex label matching.
```

Comment:
```suggestion
```

Added this to the introduction where we introduce the difference among all options.